EIPS/eip-2980.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 2980
+title: ERC-20-compatible Swiss Compliant Asset Token
+author: Gianluca Perletti (@Perlets9), Alan Scarpellini (@alanscarpellini), Roberto Gorini (@robertogorini), Manuel Olivi (@manvel79)
+discussions-to: https://github.com/ethereum/EIPs/issues/2983
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-08
+requires: 20
+---
+
+## Simple Summary
+
+An interface for asset tokens (or security tokens), compliant with Swiss Law and compatible with [ERC-20](./eip-20.md).
```

Comment:
Is there a notable difference?
```suggestion
An interface for asset tokens, compliant with Swiss Law and compatible with [ERC-20](./eip-20.md).
```

Reply:
There's not a notable difference, "Asset tokens" and "Security tokens" are synonymous